IdentityService/CQRS/User/RequestResetPassword/RequestResetPasswordHandler.cs

```diff
@@ -0,0 +1,29 @@
+﻿namespace IdentityService.CQRS.User.RequestResetPassword;
+
+public class RequestResetPasswordHandler(UserManager<ApplicationUser> userManager) : IRequestHandler<RequestResetPasswordCommand, IResult>
+{
+    public async Task<IResult> Handle(RequestResetPasswordCommand request, CancellationToken cancellationToken)
+    {
+        var applicationUser = await userManager.FindByEmailAsync(request.Email);
+        if (applicationUser is null)
+        {
+            return Results.BadRequest(
+                new List<IdentityError>
+                {
+                    new()
+                    {
+                        Code = "User",
+                        Description = "This user does not exists"
```

Comment:
I'd rather prefer to have constants with error messages for each Code, for example UserErrorMessages and have all the error messages in there, in that way, you can use them in tests as well without copy-pasting a phrase everywhere or use FluentValidator and override the default messages.